packages/containers/src/List/selector.js

```diff
@@ -102,8 +103,30 @@ export function configureGetFilteredItems(configure) {
 		},
 	);
 
-	return createSelector(
-		[getSortedList, getComponentState],
-		items => items,
+	return createSelector([getSortedList, getComponentState], items => items);
+}
+
+export function configureGetPagedItems(configure) {
+	const getSortedList = configureGetFilteredItems(configure);
+
+	const getPagedList = createSelector(
+		[getSortedList, getComponentState(configure.collectionId)],
+		(items, componentState) => {
+			let results = items;
+			if (componentState) {
+				const startIndex = componentState.get('startIndex');
+				const itemsPerPage = componentState.get('itemsPerPage');
+
+				if (itemsPerPage > 0 && startIndex > 0) {
+					results = results.slice(
```

Comment:
I would like to be sure this result is cached by the selector so we don't have re-render for nothing